.zenodo.json

```diff
@@ -33,6 +33,9 @@
         }
     ],
     "communities": [
+        {
+            "identifier": "lexibank"
+        },
```

Comment:
@StasReichert you seem to be running an older version of pylexibank - which had this issue of adding redundant communities. Could you update (`pip install -U pylexibank`)?